.gitignore

```diff
@@ -1,5 +1,5 @@
 package-lock.json
-yarn.lock
```

Comment:
I am of the opinion to ignore lock files for library packages as bumping lockfile versions isn't super useful. We also probably want to (ideally) set up the CI to test on the lowest support versions of packages as well as highest, which a lockfile would just get in the way of.